### PR TITLE
Finnish geardef and 1939/1941 loadouts

### LINKED ISSUE #58: Add Finnish loadout
Add a Finnish Winter War and Continuation War loadout.

This will include new Finnish Geardefs for uniforms (regular and winter), weapons+ammo, and general Finnish geardef for backpack and vests. Check the Russian Geardefs for an example.

We also need to make a composition for them.

### PR DESCRIPTION
**Pull request by Ludof1**
- Added Finnish gearDefs

- Added Finnish Winter War 1939 loadout

- Added Finnish early Continuation War 1941 loadout

- Created Finnish 1939 and 1941 compositions

- Added Russian smoke to gearDefRusWeapons.sqf

Closes #58

### CODE REVIEW

File: missions/2PzD_Template.VR/customization/loadouts/Finnish/loadoutFin39.sqf [hunk 1]
```diff
@@ -0,0 +1,525 @@
...
+[this, F39_MedP] call Olsen_FW_FNC_GearScript;        Medic
+[this, F39_RTO] call Olsen_FW_FNC_GearScript;         Radio Operator
+
+    //Squad
+[this, F39_SL] call Olsen_FW_FNC_GearScript;          Squad Leader
+[this, F39_TL] call Olsen_FW_FNC_GearScript;          Team Leader
+[this, F39_SMG] call Olsen_FW_FNC_GearScript;         Submachine Gunner
+[this, F39_Gren] call Olsen_FW_FNC_GearScript;        Grenadier
+[this, F39_MedS] call Olsen_FW_FNC_GearScript;        Assistant Medic
+
```
Comment (DarkMessiah27) [inline missions/2PzD_Template.VR/customization/loadouts/Finnish/loadoutFin39.sqf:22]: Is there a whiteline here or is that Github fucking up the rendering?

File: missions/2PzD_Template.VR/customization/loadouts/Finnish/loadoutFin39.sqf [hunk 2]
```diff
@@ -0,0 +1,525 @@
...
+//For Riflemen
+#define F39_Weapon_Rifleman \
+        [ \
+            [ \
+                [Fin_Vest_Mosin], \
+                [Fin_Mag_Mos,1], \
+                [Fin_Weap_MosM27], \
+                [Fin_Mag_Mos,12,"vest"] \
+            ],[40], \
```
Comment (DarkMessiah27) [inline missions/2PzD_Template.VR/customization/loadouts/Finnish/loadoutFin39.sqf:49]: 40/40 chance? Shouldn't this be 50/50? 

(Note that this is functionally the same, as far as the code cares but it looks better if it's set to 50/50)

**Review by DarkMessiah27 — CHANGES_REQUESTED** [review]

File: missions/2PzD_Template.VR/customization/loadouts/Finnish/loadoutFin39.sqf (hunk 2 above)
Comment (Ludof1) [reply missions/2PzD_Template.VR/customization/loadouts/Finnish/loadoutFin39.sqf]: It's 50/50 now, I forgot to change that when I copied that over from the 41 loadout which I did first and contains 3 weapons

File: missions/2PzD_Template.VR/customization/loadouts/Finnish/loadoutFin39.sqf (hunk 1 above)
Comment (Ludof1) [reply missions/2PzD_Template.VR/customization/loadouts/Finnish/loadoutFin39.sqf]: I copied that top part from the Russian loadouts, they seem to have the same. I got it of the empty line in these loadouts

**Review by DarkMessiah27 — APPROVED** [review]

**Pull request merged**